File: URI/C _ C++/Beginner/1036.c
#include <stdio.h>
#include <math.h>

int main(void)
{
    float a, b, c, x1, x2;

    scanf("%f %f %f", &a, &b, &c);
    if((a==0) || (((b*b)-4*a*c)<0)) {
        printf("Impossivel calcular\n");
    }

    else {
        x1 = ((-b)+sqrt((b*b)-4*a*c))/(2*a);
        x2 = ((-b)-sqrt((b*b)-4*a*c))/(2*a);
        printf("R1 = %.5f\nR2 = %.5f\n", x1, x2);
    }

    return 0;
}

File: URI/Py/1036.py
import math

a,b,c = input().split()

a = float(a)
b = float(b)
c = float(c)
try:
    y = math.sqrt((b**2)-(4*a*c))
    x1 = (-b+y)/(2*a)
    x2 = (-b-y)/(2*a)
    print("R1 = %.5f" %x1)
    print("R2 = %.5f" %x2)
except (ZeroDivisionError, ValueError):
    print("Impossivel calcular")

